pallets/manta-sbt/src/lib.rs

```diff
@@ -343,17 +377,28 @@ pub mod pallet {
         #[pallet::call_index(1)]
         #[pallet::weight(<T as pallet::Config>::WeightInfo::reserve_sbt())]
         #[transactional]
-        pub fn reserve_sbt(origin: OriginFor<T>) -> DispatchResult {
+        pub fn reserve_sbt(origin: OriginFor<T>, reservee: Option<T::AccountId>) -> DispatchResult {
```

Comment:
I think the `reserveee` should also change to polkadot signature in case someone just send lots of different account to relayer.